capture-sdk/sdk/src/main/java/net/gini/android/capture/review/multipage/MultiPageReviewFragment.java

```diff
@@ -91,7 +91,7 @@
  * <li> A {@link GiniCapture} instance is required to use the {@code MultiPageReviewFragment}
  *
  * <li> Your Activity hosting this Fragment must extend the {@link androidx.appcompat.app.AppCompatActivity}
- * and use an AppCompat Theme.
+ * and use an Material3 Theme.
```

Comment:
[Request] You can replace this whole javadoc for MultiPageReviewFragment with Internal use only. It's not part of the public API anymore.

Sorry that I wasn't able to tell you that earlier. 🙈